deep_quoridor/src/agents/core/agent.py

```diff
@@ -109,7 +109,7 @@ def end_game(self, game):
         """
         pass
 
-    def get_action(self, game) -> int:
+    def get_action(self, observation, action_mask) -> int:
```

Comment:
also replay.py needs to be updated to use this signature, can you add it to this PR?